packages/sdk/src/index.ts

```diff
@@ -147,7 +156,7 @@ export function Sdk() {
           encodedCall,
           encodedCallWeight,
           overallWeight,
-          u8aToHex(keyringPair.addressRaw),
+          scheduleAs || u8aToHex(keyringPair.addressRaw),
```

Comment:
This is scheduleXcmpTaskThroughProxy. It has to use scheduleAs, right? If so, when scheduleAs is undefined, this should throw an error.